Description: Downgrade gtest from 1.7.0 to 1.6.0
 Ubuntu trusty only provides gtest-1.6.0. The newer version was required in
 6d079c1421885038d14a1f65e71c010836b6c94c for compatibility with OS X Yosemite
 so it is safe to downgrade it for our purposes.
Author:      Ricky Ramirez <rram@reddit.com>
Last-Update: 2016-08-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: folly/folly/test/Makefile.am
===================================================================
--- folly.orig/folly/test/Makefile.am	2016-08-24 12:19:34.321676702 -0700
+++ folly/folly/test/Makefile.am	2016-08-24 12:20:37.265676702 -0700
@@ -2,7 +2,7 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
-CPPFLAGS += -Igtest-1.7.0/include
+CPPFLAGS += -Igtest-1.6.0/include
 
 TESTS= \
 	sorted_vector_types_test \
@@ -18,8 +18,8 @@
 check_PROGRAMS =
 noinst_LTLIBRARIES = lib_thread_local_test.la
 
-libgtest_la_CPPFLAGS = -Igtest-1.7.0 -Igtest-1.7.0/src
-libgtest_la_SOURCES = gtest-1.7.0/src/gtest-all.cc
+libgtest_la_CPPFLAGS = -Igtest-1.6.0 -Igtest-1.6.0/src
+libgtest_la_SOURCES = gtest-1.6.0/src/gtest-all.cc
 
 if FOLLY_TESTMAIN
 libfollytestmain_la_CPPFLAGS = $(AM_CPPFLAGS) $(libgtest_la_CPPFLAGS)
@@ -27,7 +27,7 @@
 libfollytestmain_la_LIBADD = $(top_builddir)/init/libfollyinit.la $(top_builddir)/libfolly.la
 else
 libfollytestmain_la_CPPFLAGS = $(libgtest_la_CPPFLAGS)
-libfollytestmain_la_SOURCES = $(libgtest_la_SOURCES) gtest-1.7.0/src/gtest_main.cc
+libfollytestmain_la_SOURCES = $(libgtest_la_SOURCES) gtest-1.6.0/src/gtest_main.cc
 libfollytestmain_la_LIBADD = $(top_builddir)/libfolly.la
 endif
 
